Cypher Editor - define list of commands with subcommands

FylmTM opened the pull request:
PR should provide extensive list of all commands and their subcommands support by browser.

Picture:
![image](https://cloud.githubusercontent.com/assets/626772/25551456/f4864bd4-2c7c-11e7-8f02-d330b6547341.png)

apcj commented on src/browser/modules/Editor/language/consoleCommands.js:
```diff
@@ -0,0 +1,123 @@
+    name: ':config'
+  },
+  {
+    name: ':schema - display indexes and constraints'
```
` - display indexes and constraints` should be part of the description, not part of the name

apcj commented on src/browser/modules/Editor/language/consoleCommands.js:
```diff
@@ -0,0 +1,123 @@
+      { name: 'ends-with' },
+      { name: 'contains' },
+      { name: 'rest' },
+      { name: 'rest-get'},
```
Lines 35-38 are breaking the linter; they need a space before the `}`

apcj commented on src/browser/modules/Editor/language/consoleCommands.js:
```diff
@@ -0,0 +1,123 @@
+    name: ':play',
+    description: ' - loads a mini-deck with either guide material or sample data',
+    commands: [
+      { name: 'intro' },
```
There are loads more guides than this list, but I'm sure we can the missing ones in later

FylmTM replied:
@apcj I have checked all other guides in the repo.
Looks like that I added all guides that are working right now and make sense to add them.

Other ones are either not working. Or are duplicates (there 2 guides for movie graphs, 3 guides for graph concenpts).

FylmTM commented:
@apcj 
TL;DR: - I have updated code to fix linting issues and also added missing help & guides. And added some descriptions too.

----

The whole system of how commands are handled, and help topics/guides are loaded is needed to be reviewed. It is hard to pull some information out.

Also, a bunch of guides & help topics is outdated/not converted. And information in guides itself should be re-checked.

For example in:
* In `:play start` if you click on 'start learning' it will trigger `:play concepts`
* In `:help play` you can see that there is `:play graphs`

This is 2 different guides. But the content is more-or-less same (differs in details).


Ideally all commands should be defined somewhere, and be used both for execution and for editor schema definition.

oskarhane commented:
Can I merge this now @FylmTM?

apcj commented:
@oskarhane I think the DO NOT MERGE applied to a new release of cypher-editor, which happened and this now depends on.

oskarhane commented:
@apcj I think so too, just wanted to be sure. 😸

FylmTM commented:
@oskarhane 

![image](https://cloud.githubusercontent.com/assets/626772/25588053/aeeb2706-2e9e-11e7-82f9-7359e43484b6.png)
